app/routes/template/templateAppendix.js

```diff
@@ -32,6 +58,19 @@ router.use('/', async (req, res, next) => {
     });
   }
 
+  // Convert Project ID to ident in GET endpoint
+  if (req.project && req.templateAppendix && req.method === 'GET') {
```

Comment:
it doesn't look like this is done for the template id - does it happen in the middleware? what is this step for?

Reply:
Since appendix is under template, the Template middleware happens in the parent endpoint, check:
app/middleware/template.js